automation/android/espresso/src/main/kotlin/com/instructure/espresso/InstructureActivityTestRule.kt

```diff
@@ -40,4 +40,10 @@ abstract class InstructureActivityTestRule<T : Activity>(activityClass: Class<T>
         }
     }
 
+    // Override finishActivity() so that it doesn't call finishAllActivities().  Since finishAllActivities() is called in beforeActivity(),
+    // this unnecessarily slows down the tests.
+    override fun finishActivity() {
+        //finishAllActivities()
+    }
```

Comment:
We could consider merging FinishingActivityTestRule and InstructureActivityTestRule, since the latter is the only inheritor of the former.

Reply:
The only reason FinishingActivityTestRule exists is to work around a bug in espresso core which has since been resolved.